### DISCUSSION TITLE
How to install in WSL openSUSE-Tumbleweed

### DISCUSSION THREAD

**Discussion by harshv5094** [jesseduffield/lazygit#2726] [General]
I'm using openSUSE-Tumbleweed WSL as my main development distro, I can use lazygit in powershell. But I don't know how to install lazygit in openSUSE-Tumbleween. Please tell me how to install the lazy git.

**Comment by achojo** [comment]
https://github.com/jesseduffield/lazygit#ubuntu
Although this section titled as Ubuntu, but this snippet of commands going to work for most x86_64 distros.

Of course you also have this option
https://github.com/jesseduffield/lazygit#go
but for this you'll need to install go.

**Comment by sbradnick** [comment]
I build it for myself in OBS, but you could add this repo and install from it: [devel:languages:go/lazygit](https://build.opensuse.org/package/show/devel:languages:go/lazygit) or just [grab the RPM](https://software.opensuse.org/package/lazygit?search_term=lazygit).
